docs/pages/docs/guides/index.tsx

```diff
@@ -72,6 +72,10 @@ export default function Docs() {
           Virtual fields offer a powerful way to extend your GraphQL API. This guide introduces the
           syntax and shows you how start simply and end up with a complex result.
         </Well>
+        <Well grad="grad2" heading="Images and Files Guide" href="/docs/guides/images-and-files">
+          Learn how to use the File and Image fields in Keystone to store and manage images and
+          files
+        </Well>
```

Comment:
```suggestion
        <Well grad="grad2" heading="Images and Files" href="/docs/guides/images-and-files">
            Learn how to store and manage Images and Files in Keystone.
        </Well>
```

Reply:
As a general rule, in the index pages we should avoid using "guide" or "example" in the cards as it's redundant.

However, it's probably a good idea to include "Guide:", "Example:" in the title of cards when we're in the "related resources" section of a single guide.

Reply:
Now I see it, we could also change: 

> Custom Fields ~Guide~
> Virtual Fields ~Guide~

on this page